.github/workflows/build.yml

```diff
@@ -54,4 +54,4 @@ jobs:
       run: mvn -B formatter:validate install --file pom.xml
 
     - name: Build in native
-      run: mvn -B -Pnative verify --file integration-tests/pom.xml
+      run: mvn -B -Pnative-image verify --file integration-tests/pom.xml
```

Comment:
You're right that this is wrong, but this is the wrong fix. Please use the `native` property to enable the profile: `-Dnative`

Reply:
👍